test/e2e/apps/disruption.go

```diff
@@ -215,6 +197,34 @@ var _ = SIGDescribe("DisruptionController", func() {
 			}
 		})
 	}
+
+	It("Allows updates to pdb's", func() {
+		By("Creating a pdb that targets all three pods in a test replica set")
+		createPDBMinAvailableOrDie(cs, ns, intstr.FromInt(3))
+		createReplicaSetOrDie(cs, ns, 3, false)
+
+		By("First trying to evict a pod which shouldn't be evictable")
+		pod, err := locateARunningPod(cs, ns)
+		Expect(err).NotTo(HaveOccurred())
+
+		e := &policy.Eviction{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      pod.Name,
+				Namespace: ns,
+			},
+		}
+		waitForPodsOrDie(cs, ns, 3) // make sure that they are running and so would be evictable with a different pdb
+		err = cs.CoreV1().Pods(ns).Evict(e)
```

Comment:
move this definition of `e` near Evict ?

Reply:
+1

Reply:
moved as requested